gtda/mapper/utils/visualization.py

```diff
@@ -134,8 +134,13 @@ def _get_node_colors(data, is_data_dataframe, node_elements,
             else:
                 color_data = data[:, color_variable]
         node_colors = get_node_summary(node_elements, color_data,
+
                                        summary_stat=node_color_statistic)
 
+        # normalise node colours in range [0,1] for colorscale
+        node_colors = (node_colors - np.min(node_colors)) / \
+            (np.max(node_colors) - np.min(node_colors))
+
```

Comment:
I believe that this block is too indented and is executed only when `not is_node_colors_ndarray`, while I believe it should always be.

Same question for the above: should we `get_node_summary` only in the `else` case?

Reply:
Good catch - thanks! I've fixed the indentation in cc2d646

Regarding `get_node_summary` it is correct to only call it in the `else` case as we distinguish between scenarios where the user passes an `ndarray` vs [callable, transformer, etc]. In the latter case we define `color_data` which is then used by `get_node_summary` to calculate `node_colors`

Reply:
thanks for clarifying!